data/en/lms_2.json

```diff
@@ -12190,7 +12190,7 @@
                                     }]
                                 },
                                 {
-                                    "value": "Many jobs or businesses have usual hours. Not including your overtime, how many hours do you <em>usually</em> work in your job or business?"
+                                    "value": "Many jobs or businesses have usual hours. Not including your overtime, how many hours do you <em>usually</em> work a week in your job or business?"
```

Comment:
Presumably they've checked that the translation of this is correct?

Reply:
Good spot! Yes they have confirmed it was already in the Welsh.